src/ddmd/func.d

```diff
@@ -3178,7 +3168,10 @@ extern (C++) final class UnitTestDeclaration : FuncDeclaration
 
     extern (D) this(Loc loc, Loc endloc, StorageClass stc, char* codedoc)
     {
-        super(loc, endloc, unitTestId(loc), stc, null);
+        // Id.empty can cause certain things to fail, so we create a
+        // temporary one here that serves for most purposes with
+        // createIdentifier. There is no scope to pass so we pass null.
```

Comment:
It makes me uneasy to start with one identifier, and finish with another. What fails with an empty identifier?

Reply:
I didn't like having to do it either, my first attempt was to use `Id.empty`, but when I did that the unit tests failed. I can't remember which ones or why now, but they do.